usr/lib/udev/rules.d/60-net.rules

```diff
@@ -0,0 +1 @@
+ACTION=="add", SUBSYSTEM=="net", DRIVERS=="?*", ATTR{type}=="1", PROGRAM="/usr/bin/ifcfg-devname", RESULT=="?*", NAME="$result"
```

Comment:
Also it would be better to place ifcfg-devname binary into `/lib/udev/`

```suggestion
ACTION=="add", SUBSYSTEM=="net", DRIVERS=="?*", ATTR{type}=="1", PROGRAM="/lib/udev/ifcfg-devname", RESULT=="?*", NAME="$result"
```